lighthouse-core/test/audits/viewport-test.js

```diff
@@ -33,9 +35,17 @@ describe('Mobile-friendly: viewport audit', () => {
     }).rawValue, false);
   });
 
```

Comment:
maybe add a better failure case than just not having a viewport meta tag? e.g. a non-mobile friendly but valid viewport

Also is it worth adding a test for an invalid property and/or value and test to make sure it's in the `debugString`?